src/components/ui/Link/README.md

```diff
@@ -4,13 +4,62 @@ A `<Link>` provides a way of navigating to other content, either on the same sit
 import { Paragraph } from '@octopusthink/nautilus';
 
 <Paragraph>
-  Fill out the <Link to="/sign-up">sign up form</Link> to create your account.
+  Fill out the <Link href="#sign-up">sign up form</Link> to create your account.
 </Paragraph>
 ```
 
 ## Usage
 
-Use a `<Link>` component where you would usually use an `<a>` tag in HTML. If you use the `href` prop, this link will behave exactly like an `<a>` tag; if you use the `to` prop instead, this component will use `react-router-dom`'s `Link` component to output a link that your app can use to navigate to other routes you have defined.
+Use a `<Link>` component where you would usually use an `<a>` tag in HTML.
+
+By default, Nautilus will output an `<a>` tag for every `Link` component. If you want to use a custom `Link` component throughout your app you can use the `LinkComponent` prop in your `Nautilus` higher-order component. Here's how you would use various link components from popular router libraries:
```

Comment:
This section is a bit heavy for the Usage section—I'd recommend moving it somewhere further below so it isn't as prominent and doesn't distract from core usage guidelines. Maybe under "Behaviour"? Or do we need a specific top-level section in there for developer-specific guidelines like these?

Reply:
I thought the same—it's a bit heavy for here.

Realistically this should be in a sort of "config/global settings/getting started" section. At any rate I'll move it out of here and find a better spot for it.

Reply:
Hm, good call. I wonder if the "Introduction" section should have some sub-pages—I can think of a few things that might be good to include here, notably component status & roadmap, but also we can include some getting started docs as well.

Mineral's is pretty good: https://mineral-ui.com/getting-started
Evergreen: https://evergreen.segment.com/get-started/introduction
Carbon's is better though, especially in that it offers "getting started" for both designers and developers: https://www.carbondesignsystem.com/getting-started/about-carbon
Polaris: https://polaris.shopify.com/components/get-started#navigation

It sounds like we should make a new issue for this, and maybe convert "Introduction" into a more robust "Get started" section.